exercises/01-composite-ui/before/Divergent.Customers.API/packages.config

```diff
@@ -1,28 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr" version="3.5.0.2" targetFramework="net461" />
-  <package id="Castle.Core" version="3.3.3" targetFramework="net461" />
-  <package id="Castle.Windsor" version="3.3.0" targetFramework="net461" />
   <package id="EntityFramework" version="6.1.3" targetFramework="net461" />
   <package id="Microsoft.AspNet.Cors" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net461" />
   <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.4" targetFramework="net461" />
   <package id="Microsoft.AspNet.WebApi.Cors" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Owin" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.OwinSelfHost" version="5.2.4" targetFramework="net461" />
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net461" />
-  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.7" targetFramework="net461" />
-  <package id="Microsoft.Net.Compilers" version="2.3.2" targetFramework="net461" developmentDependency="true" />
   <package id="Microsoft.Owin" version="3.1.0" targetFramework="net461" />
-  <package id="Microsoft.Owin.Cors" version="3.1.0" targetFramework="net461" />
-  <package id="Microsoft.Owin.Host.SystemWeb" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Diagnostics" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Host.HttpListener" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Hosting" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.SelfHost" version="3.1.0" targetFramework="net461" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net461" />
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net461" />
```

Comment:
Is reverting back to this old version intended?

Reply:
No. I just reverted it/upgraded json to 10 in all API projects.